src/ORM/Sluggable/SluggableSubscriber.php

```diff
@@ -93,4 +103,27 @@ private function isSluggable(ClassMetadata $classMetadata)
             $this->isRecursive
         );
     }
+
+    /**
+     * @param $entity
+     * @param EntityManager $em
+     * @param ClassMetadata $classMetadata
+     */
+    private function generateUniqueSlugFor($entity, EntityManager $em, ClassMetadata $classMetadata)
+    {
+        $repo = $em->getRepository($classMetadata->name);
+        if (!$repo instanceof SluggableRepository) {
+            // Create default repository if entity one do not extends it
+            $repo = new SluggableRepository($em, $classMetadata);
```

Comment:
I wouldn't add this feature in the first place. I think unique urls are resolvable by adding `/{id}/{slug}`.